test-conclave-init.sh

```diff
@@ -11,11 +11,11 @@ echo
 
 pushd build
 
-conclaveInitJar=$(find repo/com/r3/conclave/conclave-init/ -name 'conclave-init-*.jar')
+conclaveInitJar=$(find repo/com/r3/conclave/conclave-init/ find -name 'conclave-init-*jar' -not -name 'conclave-init-*javadoc.jar' -not -name 'conclave-init-*-sources.jar')
```

Comment:
As we are generating javadocs and sources now, this script needs more filtering, to actually find, for example,  `conclave-init-1.3.jar`